openmetadata-service/src/main/java/org/openmetadata/service/OpenMetadataApplication.java

```diff
@@ -233,7 +235,6 @@ public void run(OpenMetadataApplicationConfig catalogConfig, Environment environ
 
     registerMicrometerFilter(environment, catalogConfig.getEventMonitorConfiguration());
 
-    initializeWebsockets(catalogConfig, environment);
```

Comment:
any reason to remove this?

Reply:
moved it up, since activityFeedPublisher gets registered from resources and utilises Websocket instance